src/classes/ADDR_Validator.cls

```diff
@@ -73,14 +75,18 @@ public with sharing class ADDR_Validator {
 	                        //updating the addresses with the verified version
 	                        update validator.verifyRecords(avList, settings);
 	                    } else {
-	                        UTIL_Debug.debug(LoggingLevel.WARN, '****The class does not implement the required interface.');
+	                        ERR_Handler.processError(new ValidationException(
+	                           'The address validation class does not implement the required interface.'), 
```

Comment:
we should never be hardcoding any message strings in our code.  we need to use custom labels so NPSP can be translated fully.  I won't hold up this merge for this, but please add a task to address this.  thanks!